src/component/application/ApplicationList/ApplicationList.tsx

```diff
@@ -0,0 +1,63 @@
+import { useMemo, useState } from 'react';
+import { CircularProgress } from '@material-ui/core';
+import { Warning } from '@material-ui/icons';
+
+import { AppsLinkList, styles as commonStyles } from '../../common';
+import SearchField from '../../common/SearchField/SearchField';
+import PageContent from '../../common/PageContent/PageContent';
+import HeaderTitle from '../../common/HeaderTitle';
+import useApplications from '../../../hooks/api/getters/useApplications/useApplications';
+
+const ApplicationList = () => {
+    const { applications } = useApplications();
+    const [filter, setFilter] = useState('');
+
+    const filteredApplications = useMemo(() => {
+        const regExp = new RegExp(filter, 'i');
+        return filter
+            ? applications?.filter(a => regExp.test(a.appName))
+            : applications;
+    }, [applications, filter]);
+
+    const RenderNoApplications = () => (
```

Comment:
lowercase this, we don't want to create react components inside of other react components.